tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/contact/EmailAddressContact.scala

```diff
@@ -29,21 +34,40 @@ case class AccountEmailContact(accountId: String, id: UUID, address: MailAddress
 trait EmailAddressContactSearchEngine {
   def index(accountId: AccountId, address: MailAddress): Publisher[EmailAddressContact]
 
+  def index(accountId: AccountId, address: MailAddress, firstname: String, surname: String): Publisher[EmailAddressContact]
+
+  def index(domain: Domain, address: MailAddress, firstname: String, surname: String): Publisher[EmailAddressContact]
```

Comment:
I thought it was making sense to keep a personal index contact without names as I think it can be the case (we only type a mail address when composing a mail without a name associated to it)

However, I thought for domain likely not, as we fetch that from the ldap. But maybe there is cases when there is no name declared in a ldap? If you think yes, I could add back an extra method for domain and no name indexing

Reply:
IMO uneeded.